src/components/Book.js

```diff
@@ -1,33 +1,64 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux/es/hooks/useDispatch';
+import CircularProgress from '@mui/material/CircularProgress';
 import { RemoveBook } from '../redux/books/books';
+import styles from '../styles/Book.module.css';
 
 const Book = (props) => {
   const dispatch = useDispatch();
-  const {
-    id, title, author, category,
-  } = props;
+  const { book } = props;
   const RemoveHandler = () => {
-    dispatch(RemoveBook(id));
+    dispatch(RemoveBook(book.item_id));
   };
 
   return (
-    <div>
-      <h2>{title}</h2>
-      <p>{author}</p>
-      <p>{category}</p>
-      <button type="button">Comments</button>
-      <button type="button" onClick={RemoveHandler}>Remove</button>
-      <button type="button">Edit</button>
+    <div className={styles.BookContainer}>
+      <div className={styles.BookDetails}>
+        <span className={styles.Category}>{book.category}</span>
+        <h2 className={styles.Title}>{book.title}</h2>
+        <h3 className={styles.Author}>{book.author}</h3>
+        <div className={styles.BookButtons}>
+          <button type="button">Comments</button>
+          <button type="button" onClick={RemoveHandler}>
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className={styles.ProgressPercentage}>
+        <div id="Circle">
+          <CircularProgress variant="determinate" value={book.progress} />
+        </div>
+        <div className={styles.ProgressDiv}>
+          <span className={styles.PercentCompleted}>
+            {book.progress}
+            %
+          </span>
+          <span className={styles.Completed}>Completed</span>
+        </div>
+      </div>
+      <div className={styles.LineBook} />
+      <div className={styles.UpdateProgress}>
+        <span>Current Chapter</span>
+        <h2>
+          Chapter
+          {book.current_chapter}
+        </h2>
+        <button type="button">Update progress</button>
+      </div>
```

Comment:
- [ ] The Chapter section should be aligned more to the right, kindly tweak your design so it looks exactly like the zeplin one

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220827_070224](https://user-images.githubusercontent.com/25227022/187017392-fbfb2eed-312b-4639-92a9-23de81718bca.png)|![Screenshot_20220827_070450](https://user-images.githubusercontent.com/25227022/187017393-35b89c73-c47e-43fe-82b4-b9c444860bfd.png)|